cr-ideas.md

```diff
@@ -0,0 +1,8 @@
+# Corbin's Random Idea's
+
+- Budget Web App using Plaid's API
+- Personal Asset Management System (copyright, Corbin Roth 2022)
+    - Think of a personal form builder centered around the things you own (assets). For example, what year is your vehicle? VIN#? Wheel Size? These are all pieces of info we need...but don't need all the time. I need my cars wheel size when replacing a tire... I need my vent size when ordering new filters from Amazon etc.
```

Comment:
This would be nice...I do this now in the notes app. But it would be awesome to have an app dedicated to it instead of a section of the notes. And dedicated categories to make it easier to find things.